src/action.ts

```diff
@@ -28,6 +32,17 @@ export async function run(): Promise<void> {
 		});
 		// Set the built releaseId in the output
 		core.setOutput('release_id', releaseId);
+
+		if (releaseId) {
+			rawVersion = await balena.getReleaseVersion(parseInt(releaseId, 10));
+		}
```

Comment:
Don't have to check if the `releaseId` is set because the push command would have thrown an exception if it didn't